aspnetcore/includes/reservedRouting.md

```diff
@@ -0,0 +1,30 @@
+There are also certain sets of keywords that ASP.NET Core MVC uses internally, which should be avoided in user-code as it can result in unexpected outcomes in certain scenarios.
```

Comment:
We could even boost that to
```suggestion
There are also certain sets of keywords that ASP.NET Core MVC uses internally, which should ***not*** be used in user-code as it can result in unexpected outcomes in certain scenarios.
```